### PR TITLE
Added more configuration variables. Ref #10

### PR DESCRIPTION
**Pull request by hallzy**
Now we are able to configure if we want to just change the blank lines,
or just change the trailing whitespace as apposed to choosing to do
both.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,22 @@ Enable to clear multiple blank lines and trailing spaces when save files or not.
 - value `1`: Clear multiple blank lines and trailing spaces when save files automatically.
 - value `0`: Do not clear multiple blank lines and trailing spaces when save files automatically.
 
+### `g:janitor_auto_clean_up_trailing_ws_only_added`
```
Comment (KuoE0) [inline README.md]: :s/ws/space

File: README.md [hunk 2]
```diff
@@ -85,6 +101,30 @@ Enable to clear only on the current changes. Work with `g:janitor_auto_clean_up_
 - value `1`: Clear only on added lines when save files automatically.
 - value `0`: Clear ALL lines when save files automatically.
 
+### `g:janitor_auto_clean_up_trailing_ws`
```
Comment (KuoE0) [inline README.md]: :s/ws/space

File: plugin/janitor.py [hunk 3]
```diff
@@ -177,14 +177,17 @@ def clean_up_trailing_spaces_only_added(*args):
 def clean_up(cursor_row, cursor_col):
     filetype = vim.eval('&ft')
 
-    if vim.eval('g:janitor_auto_clean_up_only_added') == '1':
+    if vim.eval('g:janitor_auto_clean_up_trailing_ws_only_added') == '1':
```
Comment (kuoe0) [inline plugin/janitor.py]: s/ws/space

File: plugin/janitor.vim [hunk 4]
```diff
@@ -22,10 +22,30 @@ if !exists('g:janitor_auto_clean_up_on_write')
 	let g:janitor_auto_clean_up_on_write = 0
 endif
 
+if !exists('g:janitor_auto_clean_up_trailing_ws_only_added')
+	let g:janitor_auto_clean_up_trailing_ws_only_added = 0
```
Comment (kuoe0) [inline plugin/janitor.vim]: s/ws/space

File: README.md (hunk 1 above)
Comment (kuoe0) [inline README.md]: s/ws/space

File: README.md (hunk 2 above)
Comment (kuoe0) [inline README.md]: s/ws/space

File: plugin/janitor.vim [hunk 5]
```diff
@@ -38,6 +58,17 @@ if !exists('g:janitor_is_highlight')
 	let g:janitor_is_highlight = 0
 endif
 
+" If auto-clean-only-added is on, then so are these
+if (g:janitor_auto_clean_up_only_added == 1)
+	let g:janitor_auto_clean_up_trailing_ws_only_added = 1
```
Comment (kuoe0) [inline plugin/janitor.vim]: s/ws/space

File: plugin/janitor.vim [hunk 6]
```diff
@@ -38,6 +58,17 @@ if !exists('g:janitor_is_highlight')
...
+	let g:janitor_auto_clean_up_blank_lines_only_added = 1
+endif
+" If auto-clean is on, then so are these
+if (g:janitor_auto_clean_up == 1)
+	let g:janitor_auto_clean_up_trailing_ws = 1
```
Comment (kuoe0) [inline plugin/janitor.vim]: s/ws/space

File: plugin/janitor.vim [hunk 7]
```diff
@@ -22,10 +22,30 @@ if !exists('g:janitor_auto_clean_up_on_write')
...
+
+if !exists('g:janitor_auto_clean_up_blank_lines_only_added')
+	let g:janitor_auto_clean_up_blank_lines_only_added=0
+endif
+
 if !exists('g:janitor_auto_clean_up_only_added')
 	let g:janitor_auto_clean_up_only_added = 0
 endif
 
+if !exists('g:janitor_auto_clean_up_trailing_ws')
```
Comment (kuoe0) [inline plugin/janitor.vim]: s/ws/space

**Review by kuoe0 — COMMENTED** [review]
Hi, I prefer `space` than `ws`. Could you update all `ws` to `space`? Thank you for the contribution!! :)

**Comment by hallzy** [comment]
no problem... the commit has been updated to use `space` instead of `ws`

**Pull request merged**